projects/storefrontapp-e2e-cypress/cypress/integration/regression/product_details/product-details.core-e2e-spec.ts

```diff
@@ -17,9 +17,6 @@ context('Product details', () => {
 
 //TODO split this test in two files (one for mobile)
 context(`${formats.mobile.width + 1}p resolution - Product details`, () => {
-  before(() => {
-    cy.viewport(formats.mobile.width, formats.mobile.height);
-  });
```

Comment:
Do we really need to remove this viewport size in this PR?

Reply:
Just after `before` is called `beforeEach` setting exactly the same viewport size before each test case, so I found this step unnecessary. I decided to do this in this PR as this is the area of main goals for related ticket - add E2E test for Show reviews flow.